### ISSUE TITLE
`failOnError` not exiting task

### ISSUE THREAD

**Issue by clschnei** [adametry/gulp-eslint#88]
I have a method to compile coffeescript files to javascript. On a file change, it will opt to lint the file before outputting the compiled version.
```coffeescript
compile =
  # Compile javascript files
  scripts: (file) ->
    SRC = file.path or SCRIPT_SRC

    gulp.src(SRC, {base: 'webroot/src/'})
      .pipe(plumber())
      .pipe(gulpif(/[.](coffee|cjsx)$/, cjsx(bare: true)))
      # only lint changed files
      .pipe(gulpif(!!file.path, eslint()))
      .pipe(eslint.format())
      # prevent file output on error
      .pipe(eslint.failOnError())
      .pipe(gulp.dest(SCRIPT_DEST))
      .pipe(livereload())
```
EDIT: submitted on accident. Found issue while I was typing out the problem. Seems to be `plumber()` is catching and not exiting. Closing issue.
